tiatoolbox/cli.py

```diff
@@ -269,5 +283,60 @@ def save_tiles(
         )
 
 
+@main.command()
+@click.option(
+    "--source_input",
+    help="input path to the source image or a directory of source images",
+)
+@click.option("--target_input", help="input path to the target image")
+@click.option(
+    "--method",
+    help="Stain normlisation method to use. Choose from 'reinhard', 'custom',"
```

Comment:
For custom method, do you need to pass stain matrix?

Reply:
Yes - will add stain matrix as argument and default to `None` for other methods.

Reply:
which argument will be used for passing stain matrix?

Reply:
https://github.com/TIA-Lab/tiatoolbox/blob/feature-stainnorm-ruifrok-reinhard/tiatoolbox/cli.py#L298

Reply:
ok great. Please can you add an example how the matrix can be passed as input? I don't think string will work here.